OoyalaHeartbeatSampleApp/OoyalaHeartbeatSampleApp/ViewController.swift

```diff
@@ -8,10 +8,13 @@ import AVKit
 import AVFoundation
 import UIKit
 
-class PlayerViewController: AVPlayerViewController {
+class ViewController: UIViewController {
     
     let PING_FREQUENCY_S = 10
     let SSAI_GUID = "HeartbeatSampleTest";
+    var playerViewController = AVPlayerViewController()
+    var playerView = AVPlayer()
```

Comment:
Should this be just var playerView = null;   ?  Down below we reinitialize it using AVPlayer(url: url).  Otherwise we are creating an extra instance of AVPlayer

Reply:
Done, moved to an optional